Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -659,6 +659,9 @@
                             "orderedList"
                         ]
                     },
+                    "level": {
```

Comment:
Is this meant to be `"start"` instead of `"level"`?

Reply:
Yes! 😅 I didn't check my copy-paste all the way.